tests/daily_scraper/test_daily_scraper.py

```diff
@@ -6,309 +6,132 @@
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.win64.installer.exe'),
+    ({'platform': 'linux', 'branch': 'mozilla-central'},
+     '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.linux-i686.tar.bz2',
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.linux-i686.tar.bz2'),
+    ({'platform': 'linux64', 'branch': 'mozilla-central'},
+     '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.linux-x86_64.tar.bz2',
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.linux-x86_64.tar.bz2'),
+    ({'platform': 'mac', 'branch': 'mozilla-central'},
+     '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.mac.dmg',
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.mac.dmg'),
+    ({'platform': 'linux', 'branch': 'mozilla-central', 'extension': 'txt'},
+     '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.linux-i686.txt',
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.linux-i686.txt'),
+    ({'platform': 'win32', 'branch': 'mozilla-central', 'locale': 'it'},
+     '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.it.win32.installer.exe',
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central-l10n/firefox-27.0a1.it.win32.installer.exe'),
+    ({'platform': 'win32', 'branch': 'mozilla-central', 'locale': 'sv-SE'},
+     '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.sv-SE.win32.installer.exe',
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central-l10n/firefox-27.0a1.sv-SE.win32.installer.exe'),
+    ({'platform': 'win32', 'branch': 'mozilla-central', 'build_id': '20130706031213'},
+     '2013-07-06-03-12-13-mozilla-central-firefox-27.0a1.en-US.win32.installer.exe',
+     'firefox/nightly/2013/07/2013-07-06-03-12-13-mozilla-central/firefox-27.0a1.en-US.win32.installer.exe'),
+    ({'platform': 'win32', 'branch': 'mozilla-central', 'date': '2013-07-02'},
+     '2013-07-02-04-12-13-mozilla-central-firefox-27.0a1.en-US.win32.installer.exe',
+     'firefox/nightly/2013/07/2013-07-02-04-12-13-mozilla-central/firefox-27.0a1.en-US.win32.installer.exe'),
+    ({'platform': 'win32', 'branch': 'mozilla-central', 'date': '2013-07-02', 'build_number': 1},
+     '2013-07-02-03-12-13-mozilla-central-firefox-27.0a1.en-US.win32.installer.exe',
+     'firefox/nightly/2013/07/2013-07-02-03-12-13-mozilla-central/firefox-27.0a1.en-US.win32.installer.exe'),
+     # -p win32 --branch=mozilla-central --stub (old format)
+    ({'platform': 'win32', 'branch': 'mozilla-central', 'date': '2013-09-30', 'is_stub_installer': True},
+     '2013-09-30-03-02-04-mozilla-central-firefox-27.0a1.en-US.win32.installer-stub.exe',
+     'firefox/nightly/2013/09/2013-09-30-03-02-04-mozilla-central/firefox-27.0a1.en-US.win32.installer-stub.exe'),
+     # -p win64 --branch=mozilla-central --stub (old format)
```

Comment:
You mixed up this comment. For `win64` we never had a stub installer for the old format. This is just selecting an installer by date. So please move to line 33 now, which is before the `--extension` argument.

Reply:
I think I got little confused here that is why this came. So adding `#old file format` right before line 33 in branch of pull request and removing the above line would suffice what you asked right ?. 
Could you please help me understand what were the old format supported and the new one to elucidate me further on this so as to get a better hang of it.

Reply:
At some point we changed the file names as created for the stub installer. Those tests make sure that we can handle both the new and older format.